Sources/Amplitude/AMPStorage.m

```diff
@@ -0,0 +1,107 @@
+//
+//  File.m
+//
+//
+//  Created by Dante Tam on 6/10/21.
+//
+
+#import <Foundation/Foundation.h>
+
+#import "AMPStorage.h"
+
+@interface AMPStorage ()
+@end
+
+@implementation AMPStorage {
+    
+}
+
++ (NSString *)getAppStorageAmpDir {
+    NSArray *paths = NSSearchPathForDirectoriesInDomains(NSApplicationSupportDirectory, NSUserDomainMask, YES);
+    NSString *path = [paths firstObject];
+    NSString *bundleIdentifier = [[NSBundle mainBundle] bundleIdentifier];
+    return [NSString stringWithFormat:@"%@/%@/%@", path, @"/", bundleIdentifier];
+}
+
++ (NSString *)getDefaultEventsFile {
+    NSString *baseDir = [AMPStorage getAppStorageAmpDir];
+    NSString *path = [baseDir stringByAppendingString:@"/amplitude_event_storage.txt"];
+    return path;
+}
+
++ (NSString *)getDefaultIdentifyFile {
+    NSString *baseDir = [AMPStorage getAppStorageAmpDir];
+    NSString *path = [baseDir stringByAppendingString:@"/amplitude_identify_storage.txt"];
+    return path;
+}
+
++ (void)storeEvent:(NSString *) event {
+    NSString *path = [AMPStorage getDefaultEventsFile];
+    NSURL *url = [NSURL fileURLWithPath:path];
+    [AMPStorage storeEventAtUrl:url event:event];
+}
+
++ (void)storeIdentify:(NSString *) event {
+    NSString *path = [AMPStorage getDefaultIdentifyFile];
+    NSURL *url = [NSURL fileURLWithPath:path];
+    [AMPStorage storeEventAtUrl:url event:event];
+}
+
++ (void)storeEventAtUrl:(NSURL *)url event:(NSString *)event {
+    NSFileManager *fm = [NSFileManager defaultManager];
+    NSString *path = [AMPStorage getDefaultEventsFile];
```

Comment:
Both `storeIdentify` and `storeEvent` function will call  `storeEventAtUrl`, but they have different path. We should also consider the identify path?

Reply:
Oh yeah, the method should use the parameter `(NSURL *)url`, good catch.